meowbot/plugins/basic.py

```diff
@@ -181,3 +181,24 @@ def get_message_args(self, context: CommandContext):
             'icon_emoji': f'{Emoji.LANNYPARROT}',
             'username': 'Lannybot'
         }
+
+class Sing(SimpleResponseCommand):
+
+    name = 'sing'
+    help = '`sing`: get meowbot to sing you a song'
+
+    def get_message_args(self, context: CommandContext):
+        return {
+                'text': f'https://www.youtube.com/watch?v=4-L6rEm0rnY'
```

Comment:
Since these are just raw urls, you don't need to make these formatted string literals (f-strings). You can remove the `f` before the url in both commands